src/utils/Error.ts

```diff
@@ -0,0 +1,7 @@
+export enum Error {
+  LOAD = 'Unable to load a todo',
+  DELETE = 'Unable to delete a todo',
```

Comment:
You can make your error names more specific
```suggestion
  UNABLE_DELETE = 'Unable to delete a todo',
```